server/Arcadia.Assistant.Health/HealthActor.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace Arcadia.Assistant.Health
+{
+    using System;
+    using System.Data;
+    using System.Net;
+    using System.Threading.Tasks;
+
+    using Abstractions;
+    using Akka.Actor;
+    using CSP;
+    using CSP.Model;
+    using CSP.Vacations;
+    using Microsoft.EntityFrameworkCore;
+
+    public class HealthActor : UntypedActor, ILogReceive
+    {
+        private readonly Func<ArcadiaCspContext> contextFactory;
```

Comment:
should not be here. If you need Health implementation, put it in CSP library

Reply:
Done.